src/Observables.jl

```diff
@@ -38,6 +40,9 @@ Observable(val::T) where {T} = Observable{T}(val)
 
 observe(x::Observable) = x
 
+# These methods seem backwards: Observable should be swapped with AbstractObservable, because when I ask
+# to convert to a concrete type I expect to get that type.
+# They are also the source of an ambiguity.
```

Comment:
Makes sense! I'm not even sure if these are used :D

Reply:
As is this PR breaks AbstractPlotting at the build step, but if you delete them then everything builds. I haven't tested any farther than that.